devices/Max1704x/Max17048.cs

```diff
@@ -0,0 +1,456 @@
+            }
+
+            return flag;
+        }
+
+        /// <summary>
+        /// Checks if the voltage is high.
+        /// </summary>
+        /// <param name="clear">Optional. If set to true, clears the high voltage status bit.</param>
+        /// <returns>True if voltage is high; otherwise false.</returns>
+        public bool IsVoltageHigh(bool clear = false)
+        {
+            var status = Status;
+            var flag = (status & (byte)Registers.Max1704XStatusVh) > 0;
+            if (flag && clear)
+            {
+                ClearStatusRegBits((byte)Registers.Max1704XStatusVh << 8);
+            }
+
+            return flag;
+        }
+
+        /// <summary>
+        /// Checks if the voltage is low.
+        /// </summary>
+        /// <param name="clear">Specifies whether to clear the status register bits if the voltage is low.</param>
+        /// <returns><c>true</c> if the voltage is low; otherwise, <c>false</c>.</returns>
+        public bool IsVoltageLow(bool clear = false)
+        {
+            var status = Status;
+            var flag = (status & (byte)Registers.Max1704XStatusVl) > 0;
+            if (flag && clear)
+            {
+                ClearStatusRegBits((byte)Registers.Max1704XStatusVl << 8);
+            }
+
+            return flag;
+        }
+
+        /// <summary>
+        /// Returns whether the voltage has been reset. Optionally, can also clear the reset flag. </summary>
+        /// <param name="clear">Indicates whether to clear the reset flag.</param>
+        /// <returns>True if the voltage has been reset; otherwise, false.</returns>
+        public bool IsVoltageReset(bool clear = false)
+        {
+            var status = Status;
+            var flag = (status & (byte)Registers.Max1704XStatusVr) > 0;
+            if (flag && clear)
+            {
+                ClearStatusRegBits((byte)Registers.Max1704XStatusVr << 8);
+            }
+
+            return flag;
+        }
+
+        /// <summary>
+        /// Determines if the battery level is low.
+        /// </summary>
+        /// <param name="clear">Specifies whether to clear the low battery flag if it is set.</param>
+        /// <returns>True if the battery level is low; otherwise, false.</returns>
+        public bool IsLow(bool clear = false)
+        {
+            var status = Status;
+            var flag = (status & (byte)Registers.Max1704XStatusHd) > 0;
+            if (flag && clear)
+            {
+                ClearStatusRegBits((byte)Registers.Max1704XStatusHd << 8);
+            }
+
+            return flag;
+        }
+
+        /// <summary>
+        /// Checks if the status register has changed.
+        /// </summary>
+        /// <param name="clear">If set to true, clears the status register bits if they have changed.</param>
+        /// <returns>Returns true if the status register has changed; otherwise, false.</returns>
+        public bool IsChange(bool clear = false)
```

Comment:
IsChanging or IsChanged?

Reply:
To be honest - I have no idea. 
I've checked datasheet and found nothing about it. Yet it exists in sparkfun implementation....

Reply:
it's more to be proper English here :-) Is +  a verb = either past either continuous present.

Reply:
and here, I would say "IsChanged"

Reply:
Doesn't matter we can go like this, it's OK!

Reply:
makes sens, changed 😄